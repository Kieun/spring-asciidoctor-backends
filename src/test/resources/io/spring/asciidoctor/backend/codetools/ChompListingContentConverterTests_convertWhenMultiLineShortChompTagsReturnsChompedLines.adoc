= Test Document

[source,java]
----
Object chomp1 = /**/ new Object();
Object nonChomp1 = new Object();
Object chomp2 = /**/ new Object();
Object nonChomp2 = new Object();
----
