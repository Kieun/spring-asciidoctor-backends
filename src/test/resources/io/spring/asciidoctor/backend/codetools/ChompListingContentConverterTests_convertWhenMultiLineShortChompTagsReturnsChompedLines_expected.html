Object chomp1 = ...
Object nonChomp1 = new Object();
Object chomp2 = ...
Object nonChomp2 = new Object();